python/lsst/meas/extensions/photometryKron/kronLib.i

```diff
@@ -37,15 +37,39 @@ Interface to Kron magnitudes
 #include "lsst/afw/detection/Footprint.h"
 #include "lsst/afw/image/Utils.h"
 #include "lsst/afw/geom/Point.h"
-#include "lsst/meas/algorithms.h"
-#include "lsst/meas/extensions/photometryKron.h"
+#include "lsst/afw/math.h"
+#include "lsst/afw/image.h"
+#include "lsst/afw/detection.h"
+#include "lsst/afw/table.h"
+#include "lsst/meas/base.h"
+#define PY_ARRAY_UNIQUE_SYMBOL LSST_MEAS_BASE_NUMPY_ARRAY_API
```

Comment:
`itertools.product` should work with python 2.7.  Perhaps you were using an older version?